commands/taf.js

```diff
@@ -0,0 +1,28 @@
+const {SlashCommandBuilder} = require{'discord.js'}
+const {EmbedBuilder} = require{"discord.js"}
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('taf')
+		.setDescription('Gives TAF for a Specific Airport').addStringOption(option =>
+			option.setName('icao')
+				.setDescription('The input to echo back')
```

Comment:
Put something here for the users to know what they command is

Reply:
Fixed in the next commit